testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -258,25 +258,11 @@ def test_no_bonds(self):
         with pytest.raises(NoDataError):
             mdamath.make_whole(ag)
 
-    def test_not_orthogonal(self, universe, ag):
-        # Not an orthogonal unit cell
-
-        universe.dimensions = [10., 10., 10., 80., 80., 80]
-        with pytest.raises(ValueError):
-            mdamath.make_whole(ag)
-
     def test_zero_box_size(self, universe, ag):
         universe.dimensions = [0., 0., 0., 90., 90., 90.]
         with pytest.raises(ValueError):
             mdamath.make_whole(ag)
 
-    def test_too_small_box_size(self, universe, ag):
```

Comment:
Why removing that test?

Reply:
I don't think it's likely to happen, you either have box dimensions or not.  The chance of an incorrect box being supplied is low.

Reply:
We could keep the check & test, but it would make the algorithm slower because you need to calculate all bond lengths.  I thought it was too much of an unlikely case to slow things down for it